packages/react-components/src/components/FileDownloadCards.tsx

```diff
@@ -294,7 +296,7 @@ const DownloadIconTrampoline = (): JSX.Element => {
 };
 
 const useLocaleStringsTrampoline = (): _FileDownloadCardsStrings => {
-  // @conditional-compile-remove(file-sharing)
+  /* @conditional-compile-remove(file-sharing) */
   return useLocale().strings.messageThread;
-  return { downloadFile: '', fileCardGroupMessage: useLocale().strings.messageThread.fileCardGroupMessage };
+  return { downloadFile: '', fileCardGroupMessage: '' };
```

Comment:
can we do something like this so we don't forget what the old property used to be assigned to 
```suggestion
  let msg = '' 
  /* @conditional-compile-remove(file-sharing) */ 
  msg = useLocale().strings.messageThread.fileCardGroupMessage
  return { downloadFile: '', fileCardGroupMessage: msg };
```